### PR TITLE
display error message when error occurred

### PR DESCRIPTION
**Pull request by thanhlm91**
wrap notification service as message service
show messages using message service

### CODE REVIEW

**Comment by thanhlm91** [comment]
This pr will solve issue  #15

File: AdminGUI/scripts/components/form-create.component.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿import { Component, OnInit, OnDestroy, NgZone, ViewChild, ElementRef } from '@angular/core';
 import { Router, ActivatedRoute } from '@angular/router';
-
 import { IEmptyConstruct, IEntityDataService, IDataStructure, NotificationService, GenericFormComponent, EntityChangeService, EntityWithType } from 'basecode/core';
```
Comment (ngoquoc) [inline AdminGUI/scripts/components/form-create.component.ts]: Should `NotificationService` be removed?

**Review by ngoquoc — COMMENTED** [review]
Everything looks good to me, except for possible unnecessary imports of `NotificationService` from basecode.

File: AdminGUI/scripts/components/form-create.component.ts (hunk 1 above)
Comment (thanhlm91) [reply AdminGUI/scripts/components/form-create.component.ts]: Yes, it should.

**Pull request merged**